docs/samples/Microsoft.ML.Samples/Dynamic/ValueMappingStringToKeyType.cs

```diff
@@ -34,28 +34,20 @@ public static void Example()
             IEnumerable<SamplesUtils.DatasetUtils.SampleInfertData> data = SamplesUtils.DatasetUtils.GetInfertData();
             IDataView trainData = mlContext.Data.LoadFromEnumerable(data);
 
-            // Creating a list of keys based on the Education values from the dataset
+            // Creating a list of key-value pairs based on the Education values from the dataset
             // These lists are created by hand for the demonstration, but the ValueMappingEstimator does take an IEnumerable.
-            var educationKeys = new List<string>()
+            var educationKeyValuePairs = new List<KeyValuePair<string,string>>()
             {
-                "0-5yrs",
-                "6-11yrs",
-                "12+yrs"
-            };
-
-            // Creating a list of values that are sample strings. These will be converted to KeyTypes
-            var educationValues = new List<string>()
-            {
-                "Undergraduate",
-                "Postgraduate",
-                "Postgraduate"
+                new KeyValuePair<string,string>("0-5yrs", "Undergraduate"),
+                new KeyValuePair<string,string>("6-11yrs", "Postgraduate"),
+                new KeyValuePair<string,string>("12+yrs", "Postgraduate")
             };
```

Comment:
So you have this code here.

```csharp
var educationKeyValuePairs = new List<KeyValuePair<string,string>>()
{
    new KeyValuePair<string,string>("0-5yrs", "Undergraduate"),
    new KeyValuePair<string,string>("6-11yrs", "Postgraduate"),
    new KeyValuePair<string,string>("12+yrs", "Postgraduate")
};
```

I mean, that *works* I guess, it is technically not incorrect, but the most obvious thing that comes to mind when I hear something that implements an `IEnumerable<KeyValuePair<A, B>>` is `Dictionary<A, B>`. So this could have been written like so.

```csharp
var educationMap = new Dictionary<string, string>();
educationMap["0-5yrs"] = "Undergraduate";
educationMap["6-11yrs"] = "Postgraduate";
educationMap["12+yrs"] = "Postgraduate";
```

Then you just use `educationMap` as you use your `educationKeyValuePairs`.

Not saying you *have* to do it that way, but given that this is a sample and meant to be read, it might be nice if we kept "code oddity" to a minimum. (Though I don't insist on this, we can fix samples later. Only if you have time.)